feat(changelog): iam-added-enforce-authentication-methods-in-the-organiz 2025-11-17 (#5828)
scaleway/docs-content: changelog/november2025/2025-11-17-iam-added-enforce-authentication-methods-in-the-organiz.mdx

---
title: Enforce authentication methods in the Organization
status: added
date: 2025-11-17
category: security-identity
product: iam
---

> IAM administrators can now decide which authentication methods are permitted or forbidden for members inside the Organization

Reviewer: ```suggestion
IAM administrators can now decide which authentication methods are permitted or forbidden for an Organization's Members.
```
